presto-parser/src/main/java/com/facebook/presto/sql/tree/ExpressionTreeRewriter.java

```diff
@@ -555,6 +549,32 @@ public Expression visitFunctionCall(FunctionCall node, Context<C> context)
             return node;
         }
 
+        // Since OrderBy contains list of SortItems, we want to process each SortItem's key, which is an expression
+        private OrderBy rewriteOrderBy(OrderBy orderBy, Context<C> context)
```

Comment:
What do you mean? It's rewritten now in `rewriteOrderBy`.
Previously `OrderBy` was only part of `QuerySpecification` and `Query`, but those aren't expressions